Source/Plugins/IsacPluginGenerator/Packages/packages-lock.json

```diff
@@ -0,0 +1,296 @@
+{
```

Comment:
Does this file need to be committed?

Reply:
Unity started auto-generating it to match the packages in manifest.json, but I can revert and add to the gitignore if you'd prefer. I think it'll just be regenerated on the next launch anyway

Reply:
Did some digging. Looks like it's best to keep it. https://forum.unity.com/threads/packages-lock-json.912602/